crates/bevy_reflect/src/lib.rs

```diff
@@ -24,6 +24,8 @@ mod impls {
 }
 
 pub mod serde;
+pub mod std_traits;
```

Comment:
I like this module. It makes it obvious where to find `std`-based `Reflect***` structs. 

Though, imo this module should also be put in the prelude since I can see them being used a lot (as far as reflection usage goes at least haha).